cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -404,30 +404,60 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             return parseInt(this.$('#due_in').val());
         },
 
+        showProjectedDate: function() {
+            var startDate = DateUtils.parseDateFromString(this.model.get('start'))
+            this.$("#startDate").text(gettext(startDate.toDateString()));
```

Comment:
IDs must be unique in a HTML document. Are we positive these all fit that criteria? 

Also please use consistent naming conventions with these (`startDate` here, but `due-num-weeks-warning-max` below)